doc/source/data/key-concepts.rst

```diff
@@ -33,153 +26,93 @@ Since a Dataset is just a list of Ray object references, it can be freely passed
-
-See the :ref:`Creating Datasets guide <creating_datasets>` for details on how to read
-data into datasets.
+The parallelism can also be manually specified, but the final parallelism for a read is always capped by the number of files in the underlying dataset. See the :ref:`Creating Datasets Guide <creating_datasets>` for an in-depth guide
+on creating datasets.
 
 Dataset Transforms
 ==================
 
-Datasets use either Ray tasks or Ray actors to transform datasets (i.e., for
-:meth:`ds.map_batches() <ray.data.Dataset.map_batches>`,
-:meth:`ds.map() <ray.data.Dataset.map>`, or
-:meth:`ds.flat_map() <ray.data.Dataset.flat_map>`). By default, tasks are used (``compute="tasks"``). Actors can be specified with ``compute="actors"``, in which case an autoscaling pool of Ray actors will be used to apply transformations. Using actors allows for expensive state initialization (e.g., for GPU-based tasks) to be re-used. Whichever compute strategy is used, each map task generally takes in one block and produces one or more output blocks. The output block splitting rule is the same as for file reads (blocks are split after hitting the target max block size of 2GiB):
+Datasets can use either Ray tasks or Ray actors to transform datasets. By default, tasks are used. Actors can be specified using ``compute=ActorPoolStrategy()``, which creates an autoscaling pool of Ray actors to process transformations. Using actors allows for expensive state initialization (e.g., for GPU-based tasks) to be cached:
 
 .. image:: images/dataset-map.svg
-   :width: 650px
    :align: center
-
 ..
-  https://docs.google.com/drawings/d/1MGlGsPyTOgBXswJyLZemqJO1Mf7d-WiEFptIulvcfWE/edit
+  https://docs.google.com/drawings/d/12STHGV0meGWfdWyBlJMUgw7a-JcFPu9BwSOn5BjRw9k/edit
 
-See the :ref:`Transforming Datasets guide <transforming_datasets>` for an in-depth guide
+See the :ref:`Transforming Datasets Guide <transforming_datasets>` for an in-depth guide
 on transforming datasets.
 
 Shuffling Data
 ==============
 
-Certain operations like :meth:`ds.sort() <ray.data.Dataset.sort>` and
-:meth:`ds.groupby() <ray.data.Dataset.groupby>` require data blocks to be partitioned by value. Datasets executes this in three phases. First, a wave of sampling tasks determines suitable partition boundaries based on a random sample of data. Second, map tasks divide each input block into a number of output blocks equal to the number of reduce tasks. Third, reduce tasks take assigned output blocks from each map task and combines them into one block. Overall, this strategy generates ``O(n^2)`` intermediate objects where ``n`` is the number of input blocks.
+Certain operations like *sort* or *groupby* require data blocks to be partitioned by value, or *shuffled*. Datasets uses tasks to implement distributed shuffles in a map-reduce style, using map tasks to partition blocks by value, and then reduce tasks to merge co-partitioned blocks together.
```

Comment:
Can we keep those hyper links?

Reply:
Nack--- this is a key concepts guide, not a cross reference page. Adding unnecessary links and highlights, etc. harms the readability of the page by adding unnecessary emphasis.

This page was *really* hard to scan before due to all the random red, blue, etc. text highlights.